pkg/controller/plan/kubevirt.go

```diff
@@ -648,6 +648,17 @@ func (r *KubeVirt) createPodToBindPVCs(vm *plan.VMStatus, pvcNames []string) err
 			},
 		})
 	}
+	resourceReq := core.ResourceRequirements{}
```

Comment:
maybe it should be only for v2v flow? i think it is good anyway

Reply:
makes sense to always use it

Reply:
you mean to require kvm and kubevirt.io/schedulable" = "true" also when importing from ovirt/openstack?
if that's the case then I'm not so sure - if we don't need virtualization and we can't tell if the vm would start on that node later on (or even if the vm would start anytime soon) then why limiting us to such nodes?

and regardless, @liranr23 you're missing the part that sets this on the pod spec, right?

Reply:
why wouldn't we need virtualization? I thought this would allow us to start the VM where the PVC is bound too, no?

Reply:
yeah, theoretically, but we don't require the same amount of memory / cpu and potentially other resources that the vm may require so if the vm is started on the same node later on it could save us from bounding the PVC(s) on that node.. that's different than when virt-v2v is used because the conversion pod is started right after this PVC-consumer pod is starts during the migration

Reply:
ack

Reply:
you are right about the resources. i made it v2v only.